sdk/storage/azure-storage-file-share/azure/storage/fileshare/_file_client.py

```diff
@@ -519,21 +519,15 @@ def start_copy_from_url(self, source_url, **kwargs):
 
         :param str source_url:
             Specifies the URL of the source file.
-        :param file_permission_copy_mode: Specifies the option to copy file
-            security descriptor from source file or to set it using the value which is
-            defined by the value of file_permission or file_permission_key.
-            Possible values include: 'source', 'override'. Introduced in API version '2019-07-07'.
-        :param str file_permission: If specified the permission (security
-            descriptor) shall be set for the directory/file. This setting can be
-            used if Permission size is <= 8KB, otherwise file_permission_key
-            shall be used. Default value: Inherit. If SDDL is specified as
-            input, it must have owner, group and dacl. Note: Only one of the
-            file_permission or file_permission_key should be specified.
-        :param str file_permission_key: Key of the permission to be set for the
-            directory/file. Note: Only one of the file_permission or
-            file_permission_key should be specified.
-        :param ~azure.storage.fileshare.CopyFileSmbInfo copy_file_smb_info:
+        :keyword file_smb_properties:
             Additional parameters for the operation
+        :paramtype file_smb_properties: ~azure.storage.fileshare.FileSmbProperties or dict
+        :keyword bool ignore_read_only:
+            Specifies the option to overwrite the target file if it already exists and has read-only attribute set.
+        :keyword bool set_archive_attribute:
```

Comment:
Can we merge these two keyword args into smb properties class?
BTW Have we decided the name for FileSMBProperties?

Reply:
Yes I would be okay with merging them in.
I haven't had any response on the name of `FileSmbProperties` - however I think this is the name that the other languages SDKs are using - so we should probably align.

Reply:
Object removed.